### ISSUE TITLE
[FVT]:enhancement: give correct postscript and postbootscript when the template is  onieswitch

### ISSUE THREAD

**Issue by tingtli** [xcat2/xcat-core#2373]
Use --template to create a switch template and get following definition
```
[root@fs4 install]# mkdef testswitch --template onieswitch
1 object definitions have been created or modified.

[root@fs4 install]# lsdef -l testswitch
Object name: testswitch
    groups=switch
    mgt=switch
    netboot=onie
    nodetype=switch
    postbootscripts=otherpkgs
    postscripts=syslog,remoteshell,syncfiles
    switchtype=onie
    usercomment=the template for cumulus switch definition
```
Since we don't support switch to use otherpkgs and syslog,remoteshell and syncfiles at the moment. How about make the value blank. then if users use the definition, they could not wondering why they need to remove it.

**Comment by immarvin** [comment]
fix in https://github.com/xcat2/xcat-core/pull/2381

**Comment by tingtli** [comment]
Fixed and verified on xCAT 2.13.1 GA release
